Common/Securities/Cash.cs

```diff
@@ -179,7 +180,7 @@ public Security EnsureCurrencyDataFeed(SecurityManager securities, SubscriptionM
             var currencyPairs = Currencies.CurrencyPairs.Select(x =>
             {
                 // allow XAU or XAG to be used as quote currencies, but pairs including them are CFDs
-                var securityType = Symbol.StartsWith("X") ? SecurityType.Cfd : SecurityType.Forex;
+                var securityType = Symbol.StartsWith("XAU") ? SecurityType.Cfd : SecurityType.Forex;
```

Comment:
I think this drops XAG as quote currency, might need second If statement.

Reply:
Fixed